### DISCUSSION TITLE
Is there a way to duplicate the subliminal messages?

### DISCUSSION THREAD

**Discussion by ctrlalt-666** [silveredgold/beta-protection#9] [Q&A]
I want multiple subliminal messages to appear at once. Is this possible if so how?

**Comment by silveredgold** [comment]
Currently no, as it causes a nightmare for layout. As an example, if two long messages are chosen for the same message "window" not only do they overlap on anything other than very large views, you end up pretty much drawing over the entire viewable window, which causes more issues: messages would either extend off the ends of the screen, or I'd have to make the font size dynamic based on the combined length of multiple chosen messages.

In short, its one of those surprisingly complex issues so I don't see this feature being added. If someone else contributes a reasonably reliable implementation, I'll happily include it, but its not something I'll be adding.
